practices/devops.md

```diff
@@ -93,10 +93,10 @@ Sharing is crucial. If the teams share responsibilities, they share success as w
 - Long feedback loops
 - Long incident resolution times
 
-**Resources for the Devops**
+## Resources for the Devops ##
```

Comment:
@davidfuchsa the MD headline is only at the beginning, not at the end

Reply:
Should be ok now